docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -1225,14 +1293,28 @@ code: |
     "sources": []
   }
   
-  create_package_zip(package_title,
-        interview.package_info(
+  package_info = interview.package_info(
           get_pypi_deps_from_choices(
             interview.jurisdiction_choices.true_values() +
             interview.org_choices.true_values()
           )
-        ),
-        {"author name and email":"author@example.com"},
+        )    
+  
+  if interview.author and str(interview.author).splitlines():
+    # TODO(qs): is it worth ever adding email here?
+    # It would conflict with listing multiple authors
+    default_vals = {
+      "author name and email": str(interview.author).splitlines()[0]
+    }
+    package_info['author_name'] = default_vals['author name and email']
```

Comment:
If you add email, the following also needs to be handled, which could be of significant volume: 
          `If author is set to Court Forms Online`

Also what if the author's email changed, like when the students left the school?

I guess if email is really useful, you can add a placeholder for it instead of the actual value, but I am not sure how you intend to use it, and how many authors are willing to accept that.

Reply:
From what I remember, I thought we wanted to avoid including direct emails in the interviews, due to the fact the users sometimes email us asking for help and assistance on cases that we can't give.

Reply:
The field is named "author name and email" but I agree, we don't want to actually put an email here. I believe this mirrors the name that Docassemble is looking for for either the setup.py or the "Packages" menu input field.